examples/cpp/pyperf/PyPerfType.h

```diff
@@ -224,6 +231,9 @@ struct PyPerfSample {
   uint8_t stackStatus;
   int32_t kernelStackId;
   std::vector<int32_t> pyStackIds;
+  uint64_t userIp;
+  uint64_t userSp;
```

Comment:
Are these used anywhere besides `NativeStackTrace`? If not, no reason to store them here instead of in `NativeStackTrace`